deploy/rbd/docker/Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM centos:7
-LABEL maintainers="Kubernetes Authors"
-LABEL description="RBD CSI Plugin"
-
-ENV CEPH_VERSION "mimic"
-RUN yum  install -y centos-release-ceph && \
-    yum install -y ceph-common e2fsprogs xfsprogs rbd-nbd && \ 
-    yum clean all
-
-COPY rbdplugin /rbdplugin
-RUN chmod +x /rbdplugin
-ENTRYPOINT ["/rbdplugin"]
+FROM ceph/ceph:v14.2
```

Comment:
Can we make FROM an ARG?

Then we can have it build multiple versions of the container, one for each major version of ceph.

Reply:
Here's an example of how that can work:
https://github.com/pnnl-miscscripts/miscscripts/blob/master/containers/k8s-node-image-nginx/Dockerfile

Reply:
This is the client container and hence should work across multiple older Ceph cluster versions. Does it matter or require that we build various client versions based images?

If it does, then the deploy.sh would have to publish multiple images based on different client versions, I would look at it as a followup to this PR assuming this gets accepted.

Reply:
yeah, that is a good question. if it is completely backwards compatible, it shouldn't matter. But without some kind of testing (automated?), it might be easier just to ensure compatibility by releasing major versioned images.